src/System Application/App/Language/src/AllowedLanguages.Page.al

```diff
@@ -0,0 +1,37 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Globalization;
+
+/// <summary>
+/// Page that shows the list of allows languages which is enabled for this environment. If nothing is specified, then the user will be able to use all available languages.
+/// </summary>
+page 3563 "Allowed Languages"
+{
+    Caption = 'Allowed Languages';
+    PageType = List;
+    ApplicationArea = All;
+    UsageCategory = Administration;
+    SourceTable = "Allowed Language";
+    HelpLink = 'https://go.microsoft.com/fwlink/?linkid=2149387';
+    AdditionalSearchTerms = 'company,role center,role,language';
+    AboutTitle = 'About allowed languages.';
+    AboutText = 'Define a list of allowed languages which is enabled for this environment. If nothing is specified, then the user will be able to use all available languages.';
+    Permissions = tabledata "Allowed Language" = r;
+
+    layout
+    {
+        area(Content)
+        {
+            repeater(AllowedLanguages)
+            {
+                field("Language Id"; Rec."Language Id")
+                { }

```

Comment:
This {} on a new line does hurt my eyes a litte, but it's such a tiny thing, that I'll let it slide 😝